copyhtml/copyhtml.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * copyHTML copies the HTML of the elements specified by the user to the clipboard.
+ */
+export function copyHTML(transform?: (html: string) => string) {
```

Comment:
Refactor the `copyHTML` function to take multiple options.
```suggestion
export interface CopyHTMLOptions {
  render: (output: string) => void;
  transform?: (html: string) => string;
}

export function copyHTML(options: CopyHTMLOptions) {
```

This allows us to customize what happens to the data once it is scraped and transformed.

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/2fe0b3ba79daab609db8a79a0dc3ab0aa659e742>.